examples/Makefile-cc2538

```diff
@@ -70,11 +70,19 @@ ifeq ($(DHCP6_CLIENT),1)
 configure_OPTIONS              += --enable-dhcp6-client
 endif
 
+TopSourceDir                    := $(dir $(shell readlink $(firstword $(MAKEFILE_LIST))))..
+AbsTopSourceDir                 := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))..
+
+CONFIG_FILE      = OPENTHREAD_PROJECT_CORE_CONFIG_FILE='\"openthread-core-cc2538-config.h\"'
+CONFIG_FILE_PATH = $(AbsTopSourceDir)/examples/platforms/cc2538/
+
 COMMONCFLAGS                   := \
     -fdata-sections               \
     -ffunction-sections           \
     -Os                           \
     -g                            \
+    -D$(CONFIG_FILE)              \
+    -I$(CONFIG_FILE_PATH)         \
```

Comment:
These changes don't seem to work. I put some bad characters in code that would only compile in if the correct defines from this header were set and I don't get any compile errors. Can anyone tell me what I am doing wrong? I copied this from `Makefile-nrf52840`, but that makefile seem to get the build errors as expected.